submissionforms.md

```diff
@@ -138,6 +138,57 @@ Provide detailed information about a specific input-form. The JSON response docu
           "languageCodes": []
         }
       ]
+    },
+    {
+      "fields": [
+        {
+          "input": {
+            "relationship": true
```

Comment:
Do we really need this `"relationship": true`? Couldn't it be implicit, if there are some elements inside `selectableRelationship`?

Reply:
It can indeed be implicit, but it may also make it easier for Angular to verify whether it should verify the `selectableRelationship` section. Either would be fine for me

Reply:
I would prefer instead to have a:
` "type": "relationship"`

Reply:
The `"relationship": true` has been removed now as discussed in the meeting